app/models/concerns/itemizable.rb

```diff
@@ -74,6 +74,20 @@ def line_items_quantities
 
   private
 
+  def line_item_items_quantity_is_positive
```

Comment:
@armahillo I believe there's an existing validation for this, right?

Reply:
I did not find it anywhere.

Reply:
@pbteja1998 Looks like `LineItem` has a validation for this: https://github.com/rubyforgood/diaper/blob/126d6be93ce22dde7773e4bd177a6ef54b223f30/app/models/line_item.rb#L19 

and `InventoryItem` also:
https://github.com/rubyforgood/diaper/blob/126d6be93ce22dde7773e4bd177a6ef54b223f30/app/models/inventory_item.rb#L20

I think you're probably OK to remove this, but be sure to test for this in your Model specs :)